include/ddc/kernels/splines/matrix_center_block.hpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (C) The DDC development team, see COPYRIGHT.md file
+//
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <memory>
+#include <utility>
+
+#include <string.h>
+
+#include "matrix_corner_block.hpp"
+#include "view.hpp"
+
+namespace ddc::detail {
+class Matrix;
+
+template <class ExecSpace>
+class Matrix_Center_Block : public Matrix_Corner_Block<ExecSpace>
```

Comment:
In the rest of the code CamelCase is used for class names. Is there any reason that it isn't used here?

Reply:
No, it's residual from SLL version, I will make the change